cmd/opdoc/opdoc.go

```diff
@@ -342,7 +329,7 @@ func (nt namedType) boundString() string {
 
 // LanguageSpec records the ops of the language at some version
 type LanguageSpec struct {
-	EvalMaxVersion  int
+	EvalMaxVersion  uint64
```

Comment:
Maybe this should be renamed to just "Version" now?

Reply:
Good idea, done in 780068665